data/mods/Xedra_Evolved/recipes/inventor/researches.json

```diff
@@ -11,6 +11,7 @@
     "time": "2 h",
     "skill_used": "deduction",
     "difficulty": 2,
+    "flags": [ "BLIND_EASY" ],
```

Comment:
If it requires the laptop how is it blind easy?  Or are you saying the laptop screen lights up enough to work from and it's just a weird side effect that blind easy means you can now use your laptop with a blindfold on?

Reply:
I thought blind easy refer only to the fact you can easily craft it at night, does it really allows you to craft while you blind?

Reply:
Went ahead and tested it, BLIND_EASY allows you to craft that recipe while wearing a blindfold.

Reply:
pretty dumb, but i don't think i can do much here - i just want to bring this aestetic of "researching till dawn"